Content.Server/_Starlight/GameTicking/GameTicker.PeacefulRoundEnd.cs

```diff
@@ -5,24 +5,28 @@
 using Content.Shared.Mech.Components;
 using Robust.Server.Player;
 using Robust.Shared.Configuration;
+using Content.Shared.GameTicking;
 
 namespace Content.Server.Starlight.GameTicking;
 
 public sealed class PeacefulRoundEndSystem : EntitySystem
 {
     [Dependency] private readonly IConfigurationManager _cfg = default!;
     private bool _isEnabled = false;
+    private bool _roundedEnded = false;
 
     public override void Initialize()
     {
         base.Initialize();
         _cfg.OnValueChanged(StarlightCCVars.PeacefulRoundEnd, v => _isEnabled = v, true);
         SubscribeLocalEvent<RoundEndTextAppendEvent>(OnRoundEnded);
+        SubscribeLocalEvent<PlayerSpawnCompleteEvent>(OnSpawnComplete);
+        SubscribeLocalEvent<Content.Shared.Chemistry.Components.GotRehydratedEvent>(OnSpawnComplete);
```

Comment:
don't specify the full path to the event, use using for that.
```suggestion
        SubscribeLocalEvent<GotRehydratedEvent>(OnSpawnComplete);
```

Reply:
```suggestion
        SubscribeLocalEvent<GotRehydratedEvent>(_=>SpreadPeace());
```